src/main/java/step2/Operator.java

```diff
@@ -0,0 +1,40 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+
+    PLUS("+", (num1, num2) -> num1 + num2),
+    MINUS("-", (num1, num2) -> num1 - num2),
+    MUL("*", (num1, num2) -> num1 * num2),
+    DIV("/", (num1, num2) -> num1 / num2);
+
+    private final String op;
+    private final BiFunction<Integer, Integer, Integer> expression;
+
```

Comment:
```suggestion
private static final Map<String, Operator> valuesMap = Arrays.stream(values())
                                            .collect(Collectors.toMap(Operator::getOperator, Function.identity()));
```
위 코드의 의미는 values()로 생성한 Operator 배열을 "+", "-" 와 같이 Operator에 해당하는 String을 키로 갖고, Operator 자체를 값으로 갖는 Map을 생성하는 것 입니다.
지금 숭문님이 getOperator에서 작성해주신 것 처럼 해주셔도 되지만, 매번 Stream을 열 필요 없이 이렇게 맵을 만들 때 한번만 열어도 된다는 장점이 있죠 :)
Stream API 의 Collectors 클래스와 Function.identity() 메서드에 대해서 공부해보시면 좋을 것 같습니다.